pina/__init__.py

```diff
@@ -5,7 +5,8 @@
     'Plotter',
     'Condition',
     'Location',
-    'CartesianDomain'
+    'CartesianDomain',
+    'SimplexDomain'
```

Comment:
remove `SimplexDomain` from the pina init. The user should call 
``` from pina.geometry import SimplexDomain ```

Reply:
Fixed the __init__